### PR TITLE
Hotfix park_info.clj

### PR DESCRIPTION
**Pull request by DeC0rtez**
Applying hotfix to par-info.clj file after previous erros in versioning.

### CODE REVIEW

File: park_info.clj [hunk 1]
```diff
@@ -1,198 +1,197 @@
...
-        :opening_hours "park - year long, villa - (April 10:00-17:00, May-September 10:00-18:00, October 10:00-17:00, November-March CLOSED) "
-        ;---Map---
-        :map_url "https://www.praha.eu/public/bf/26/3e/184361_4_hvezda_mapa.pdf"
+(def Frantiskanska_Zahrada{
+    ;---Basic Info---
+    :food true
+    :food_extra "In the vicinity - Alfa passage, Wenceslaus & Jungmann square, Narodni Trida"
+    :toilet true
+    :toilet_extra "with wheelchair access"
+    :POIs "Pleasant dining in the historical town district, suitable for both children and the elderly, Church of Our Lady of the Snows"
```
Comment (Lucius1010011010) [inline park_info.clj:36]: Move the 'suitable for...' as the last point, maybe? That way it makes more sense I think: in this park there is X, Y and Z; suitable for A and B.

File: park_info.clj [hunk 2]
```diff
@@ -1,198 +1,197 @@
...
-        :opening_hours "April-October: 09:00-18:00, November-March: 09:30-16:00"
-        ;---Map---
-        :map_url "https://www.praha.eu/public/fe/bd/6a/184189_4_bertramka_mapa.pdf"
+(def Bertramka{
+    ;---Basic Info---
+    :food nil
+    :food_extra nil
+    :toilet true
+    :toilet_extra nil
+    :POIs "Cultural monument, classical music concerts, social events, W.A. Mozart Museum" ;POI refers to Point of Interest
```
Comment (Lucius1010011010) [inline park_info.clj:7]: No 'suitable for'?

File: park_info.clj [hunk 3]
```diff
@@ -1,198 +1,197 @@
...
+(def Obora_Hvezda{
+    ;---Basic Info---
+    :food true
+    :food_extra "During Summer Season on the premises of the Villa (www.pamatniknarodnihopisemnictvi.cz), tel:. +420 235 357 938,
+    or at Libocká street, restaurant U Holečků, www.uholecku.cz,
+    or at Libocká studna, tel. +420 777 836 966,
+    alternatively at Petřiny"
+    :toilet true
+    :toilet_extra "Portable toilets on the premises, during summer season toilets are also available in the villa"
+    :POIs "Suitable for tourists and families with children, heritage trees, nature trail, natural attractions"
```
Comment (Lucius1010011010) [inline park_info.clj:69]: Again, wouldn't start with 'suitable for...'.

File: park_info.clj [hunk 4]
```diff
@@ -1,198 +1,197 @@
...
+
+(def Kampa{
+    ;---Basic Info---
+    :food true
+    :food_extra "restaurant: Rybí Restaurace,
+    café: Mlýnská (tel. +420 608 444 490)
+    restaurant/café : Sovovy Mlýny"
+    :toilet true
+    :toilet_extra nil
+    :POIs "View of the river, Sovovy Mlýny gallery, Čertovka, Charles Bridge, Werich House, heritage trees"
```
Comment (Lucius1010011010) [inline park_info.clj:102]: No 'suitable for'. Either keep it everywhere, or cut it out everywhere I would think.

File: park_info.clj [hunk 5]
```diff
@@ -1,198 +1,197 @@
...
+    ;---Children and Sport---
+    :bike_path true
+    :skating true
+    :sports_field false
+    :sports_field_extra "no specialized sports facilities, but grass-covered segments available for sports"
+    :playground true
+    :playground_extra "yes, in the vicinity: Community and Maternal centre Kampa"
+    ;---Directions---
+    :public_transport "
+    stop: Helichova - tram:s #12, #20, #20, #22, #23, #57
```
Comment (Lucius1010011010) [inline park_info.clj]: Typo: 'tram:s'.

File: park_info.clj [hunk 6]
```diff
@@ -1,198 +1,197 @@
...
+    :map_url "https://www.praha.eu/public/7c/a6/ed/184308_4_kampa_mapa.pdf"       
+})
+
+(def Kinskeho_zahrada{
+    ;---Basic Info---
+    :food true
+    :food_extra "café in Musaion museum of natural sciences"
+    :toilet true
+    :toilet_extra nil
+    :POIs "Enjoyable location for a walk with children, Quiet and relaxing space in the center of town, Musaion Museum of Natural Sciences, decorative ponds, natural attraction - Petřín rock formations"
```
Comment (Lucius1010011010) [inline park_info.clj:132]: 'Suitable for' in the beginning.

File: park_info.clj [hunk 7]
```diff
@@ -1,198 +1,197 @@
...
+    :map_url "https://www.praha.eu/public/3/95/58/184325_4_kinskeho_mapa.pdf"       
+})
+
+(def Klamovka{
+    ;---Basic Info---
+    :food true
+    :food_extra "restaurant & club Klamovka"
+    :toilet true
+    :toilet_extra "in the restaurant and club"
+    :POIs "Historically significant sights, art gallery, musical club with restaurant in garden, playground for small children"
```
Comment (Lucius1010011010) [inline park_info.clj:160]: Not going to keep commenting on if from now on, but 'suitable for' should be either added at the end here and elsewhere, or cut out entirely. Maybe added as a separate variable.

File: park_info.clj [hunk 8]
```diff
@@ -209,164 +208,169 @@
...
+    ;---Children and Sport---
+    :bike_path true
+    :skating true
+    :sports_field true
+    :sports_field_extra "Sokol Vinohrady"
+    :playground true
+    :playground_extra "yes, by the restaurant"
+    ;---Directions---
+    :public_transport "
+    metro: Jiřího z Poděbrad (Line A)
```
Comment (Lucius1010011010) [inline park_info.clj]: Non-standard info display. Should be
stop: Jiřího z Poděbrad - metro line(s): A

File: park_info.clj [hunk 9]
```diff
@@ -209,164 +208,169 @@
...
+    stop: Na Smetance - bus: #135
+    stop: Italská/ Vinohradská Tržnice - tram: #11"
```
Comment (Lucius1010011010) [inline park_info.clj]: Why the space after slash?

**Review by Lucius1010011010 — CHANGES_REQUESTED** [review]
One non-standard info output; mostly comments about better structuring the outputs.

File: park_info.clj (hunk 1 above)
Comment (DeC0rtez) [reply park_info.clj:36]: These are literal translations from the website. Don't personally see a reason to change these.

File: park_info.clj (hunk 5 above)
Comment (DeC0rtez) [reply park_info.clj]: fixed.

File: park_info.clj (hunk 9 above)
Comment (DeC0rtez) [reply park_info.clj]: Two stops, with the same tram going to them. Gonna put them on individual lines.

File: park_info.clj (hunk 2 above)
Comment (DeC0rtez) [reply park_info.clj:7]: These are literal translations from the website. Some parks have them, some don't. I Don't personally see a reason to change these.

File: park_info.clj (hunk 3 above)
Comment (DeC0rtez) [reply park_info.clj:69]: These are literal translations from the website. Some parks have them, some don't. I Don't personally see a reason to change these.

File: park_info.clj (hunk 4 above)
Comment (DeC0rtez) [reply park_info.clj:102]: These are literal translations from the website. Some parks have them, some don't. I Don't personally see a reason to change these.

File: park_info.clj (hunk 6 above)
Comment (DeC0rtez) [reply park_info.clj:132]: These are literal translations from the website. Some parks have them, some don't. I Don't personally see a reason to change these.

File: park_info.clj (hunk 7 above)
Comment (DeC0rtez) [reply park_info.clj:160]: These are literal translations from the website. Some parks have them, some don't. I Don't personally see a reason to change these.

File: park_info.clj (hunk 8 above)
Comment (DeC0rtez) [reply park_info.clj]: fixed.

**Review by Lucius1010011010 — APPROVED** [review]
All issues fixed; approved for merge.

**Pull request merged**